src/test/java/pages/MenuPage.java

```diff
@@ -23,6 +23,8 @@ public class MenuPage {
     private final Actions action;
     private WebDriverWait wait;
 
+    private Duration duration;
```

Comment:
Where do you use this field?

Reply:
Yes, we don`t need use this field here. I will delete it.